extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -270,15 +281,17 @@ export class SchemaCompareResult {
 	private reExecute(): void {
```

Comment:
maybe this should be named something different from "reExecute" since it's the only path now to start the comparison

Reply:
Sure, I can name it startCompare() I guess...